web/src/layouts/MainLayout/MainLayout.js

```diff
@@ -47,27 +73,53 @@ const MainLayout = ({ children }) => {
           <ul className="flex items-center">
             <li className={getActiveClasses("mr-8 h-10 w-10 rounded-full border-2 border-gray-700 flex items-center justify-center", {'border-indigo-300': currentUser})}>
               <button className="h-full w-full" onClick={() => currentUser && routes.newPart2({userName: data?.user?.userName})}>
-                <Svg name="plus" className={getActiveClasses("text-gray-700 w-full h-full",{'text-indigo-300': currentUser})} />
+              <Svg name="plus" className={getActiveClasses("text-gray-700 w-full h-full",{'text-indigo-300': currentUser})} />
               </button>
             </li>
-            {
-              isAuthenticated ?
-              <li className="h-10 w-10 border-2 rounded-full border-indigo-300 text-indigo-200">
-                <a href="#" onClick={logOut}>
-                  {!loading && <ImageUploader
-                    className="rounded-full object-cover"
-                    onImageUpload={() => {}}
-                    aspectRatio={1}
-                    imageUrl={data?.user?.image}
-                    width={80}
-                  />}
-                </a>
-              </li>:
-              <li>
-                <a href="#" className='text-indigo-200 font-semibold underline' onClick={logIn}>Sign in/up</a>
-              </li>
-            }
+            <li className="h-10 w-10 border-1 rounded-full border-indigo-300 text-indigo-200">
+            <div aria-describedby={popoverId} onMouseOver={togglePopover}>
+                {!loading && <ImageUploader
+                  className="rounded-full object-cover"
+                  onImageUpload={() => {}}
+                  aspectRatio={1}
+                  imageUrl={data?.user?.image}
+                  width={80}
+                />}
+              </div>
+            </li>
           </ul>
+          <Popover
+            id={popoverId}
+            open={isOpen}
+            anchorEl={anchorEl}
+            onClose={closePopover}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'left',
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'left',
+            }}
```

Comment:
Like I said in the issue, I think thiswould be better as
```
anchorOrigin={{
    vertical: 'bottom',
    horizontal: 'right',
  }}
  transformOrigin={{
    vertical: 'top',
    horizontal: 'right',
  }}
```

Reply:
okay i will update this one